Gemfile.lock

```diff
@@ -172,4 +172,4 @@ DEPENDENCIES
   rspec (~> 3.0)
 
 BUNDLED WITH
-   2.0.1
+   2.1.2
```

Comment:
Travis was failing due to this change https://github.com/ashishprajapati/activestorage-dropbox/pull/12/commits/0ba33fd278cf1ef64fff883cff27d0f86e96a30b#diff-e79a60dc6b85309ae70a6ea8261eaf95R175. I have updated the Travis config with `2.1.2` to fix it.